src/store/modules/command_params.js

```diff
@@ -86,8 +87,36 @@ const getters = {
 
 const mutations = {
 
-    mount_ra(state, val) { state.mount_ra = val; },
-    mount_dec(state, val) { state.mount_dec = val; },
+    mount_ra(state, val) { 
+        // Clear the mount object name if the new coordinates have changed significantly
+        const large_anglular_difference_degrees = 3
+
+        let old_ra = helpers.hour2degree(state.mount_ra) // convert hours to degrees
+        let new_ra = helpers.hour2degree(val)
+        let dec = state.mount_dec // since dec doesn't change here, use it for both positions
+        let angle = helpers.angular_distance(old_ra, dec, new_ra, dec)
+        if (angle !== NaN & angle > large_anglular_difference_degrees) {
+            state.mount_object = ''
+        }
+
```

Comment:
Is this going to pose a problem when users search for another object in Aladin or click on another common target? (Meaning, is this going to clear the `mount_object` value after picking a completely new target because `large_angular_difference_degrees` is going to be large in that case?)

Reply:
Good question. The solution feels like it could be cleaner, but basically if you search for a new target, the search component simply returns the results; it doesn't set them to vuex yet or anything. So the parent component is responsible for handling the results (reasoning being that perhaps this component is useful in places unrelated to telescope command parameters). 

In order to avoid the problem you're wondering about, the parent needs to take the results and update the coordinate parameters before updating the object name. If you do it in reverse order, then the object name gets erased. 

For example, in Telescope.vue, lines 234-240:
```javascript
    handle_coordinate_search_results(search_results) {
      if (!search_results.error) {
        this.mount_ra = search_results.ra.toFixed(4);
        this.mount_dec = search_results.dec.toFixed(4);
        // make sure to change this after the coordinates, since the object name is cleared 
        // after large changes in the coordinate positions. Details in vuex command_params.
        this.mount_object = search_results.searched_name;
        ...
```

Reply:
Okay, that's kind of what I figured! So just keep in mind at this point that updating `mount_object` in the proper order should avoid that